redux-1/src/LocalStorageManager.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+
+import { getAttendingInformation } from './selectors';
+import { changeAttendingInformation } from './actions';
+
+import type { Attending } from './types/state';
+
+type Props = {|
+  +attendingInformation: Attending[],
+  +changeAttendingInformation: typeof changeAttendingInformation,
+|};
+
+class LocalStorageManager extends PureComponent<Props> {
+  componentDidMount() {
```

Comment:
mhh

Reply:
Empty initial rendering is intended as an improvement for a next version?

Reply:
en fait il n'y a pas d'empty initial rendering :)
cf la doc de React:
```
You may call setState() immediately in componentDidMount(). It will trigger an extra rendering, but it will happen before the browser updates the screen. 
```